__tests__/app.test.js

```diff
@@ -34,8 +32,37 @@ describe("GET /api/topics", () => {
       .expect(200)
       .then(({ body }) => {
         const { topics } = body;
-        expect(topics[0]).toHaveProperty("slug");
-        expect(topics[0]).toHaveProperty("description");
+        topics.forEach((topic) => {
```

Comment:
This refactor has created a test which will auto pass if the returned array is empty. We need a length check here